### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by Johasan**
Added new file BasePage

### CODE REVIEW

File: justAutomationFramework.iml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <orderEntry type="library" name="Maven: org.apache.commons:commons-lang3:3.11" level="project" />
+    <orderEntry type="library" name="Maven: org.apache.httpcomponents.client5:httpclient5:5.0.3" level="project" />
+    <orderEntry type="library" name="Maven: org.apache.httpcomponents.core5:httpcore5:5.0.2" level="project" />
+    <orderEntry type="library" name="Maven: org.apache.httpcomponents.core5:httpcore5-h2:5.0.2" level="project" />
+    <orderEntry type="library" name="Maven: commons-codec:commons-codec:1.13" level="project" />
+    <orderEntry type="library" name="Maven: org.rauschig:jarchivelib:1.1.0" level="project" />
+    <orderEntry type="library" name="Maven: org.apache.commons:commons-compress:1.20" level="project" />
+    <orderEntry type="library" name="Maven: org.jsoup:jsoup:1.13.1" level="project" />
+  </component>
+</module>
```
Comment (Znamm9) [inline justAutomationFramework.iml:48]: remove this shit from remote please @Johasan

File: src/test/java/BasePage.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BasePage {
+    WebDriver driver;
+
+    public void setUp(WebDriver driver){
+        this.driver = driver;
+    }
+
```
Comment (Znamm9) [inline src/test/java/BasePage.java:11]: remove this line please

File: src/test/java/BasePage.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BasePage {
+    WebDriver driver;
+
```
Comment (Znamm9) [inline src/test/java/BasePage.java:7]:
```suggestion
@BeforeTest
```

Add annotation please

**Review by Znamm9 — CHANGES_REQUESTED** [review]

File: src/test/java/BasePage.java (hunk 2 above)
Comment (Johasan) [reply src/test/java/BasePage.java]: Done

File: src/test/java/BasePage.java (hunk 3 above)
Comment (Johasan) [reply src/test/java/BasePage.java:7]: Changed it

**Review by Znamm9 — APPROVED** [review]

**Pull request merged**